.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Description
+
+<!-- Describe the bug this PR fixes or the feature it adds. Link to any related issues and PRs -->
+
+#### Breaking change?
+
+<!-- If applicable, list the APIs/functionality which this PR breaks -->
+
+### Example
+
+<!-- If applicable, add an example on how this improves the application -->
+
+---
+
+### Checklist
+
+- [ ] All tests pass
+- [ ] Tests added in this PR (if applicable)
+
```

Comment:
Ultimately, we will have a CI running tests + assertions on code coverage evolution 🤞🏻

Reply:
I'm happy to remove this when that's the case. 👍